hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServer.java

```diff
@@ -82,7 +82,13 @@ protected void configureServlets() {
       LOG.info("Creating Recon Schema.");
       reconSchemaManager.createReconSchema();
 
-      httpServer = injector.getInstance(ReconHttpServer.class);
+      // TODO Remove try-catch block after HDDS-2041 is fixed.
+      // Allowing Recon server start even if HTTP server start failed
+      try {
```

Comment:
One question if HttpServer start failed will any functionality of recon works, like getContainers etc.,

But whereas in OM SCM, core functionality still works even if HttpServer does not start. Not sure if I am missing something here.

Reply:
@bharatviswa504 Yes that is a valid question. On further investigation, I discovered that this may be due to jetty upgrade in hadoop in the distribution. This affects all Ozone components. Also, in the case of SCM, even though it withstands a HttpServer crash, since the initialization is stopped midway due to exception, none of the event handlers are registered thereby making the component useless.

I plan to hold on to this PR, wont commit it. If jetty version is indeed the problem, I will close this PR without merging the changes.